proto/wechaty/puppet/friendship.proto

```diff
@@ -63,6 +63,7 @@ message FriendshipSearchWeixinResponse {
 message FriendshipAddRequest {
   string contact_id = 1;
   string hello = 2;
+  string options = 3;
```

Comment:
We should create a message type for this options, instead of using a string here.